munch-match/src/components/FoodOptionButton.jsx

```diff
@@ -1,8 +1,10 @@
+import styles from "./FoodOptionButton.module.css"
+
 const FoodOptionButton = ({ foodOption, disabled, onClick }) => {
 
     return (
-        <button className={`button ${disabled ? "disabledButton" : ""}`} disabled={disabled} onClick={() => onClick(foodOption)}>
-            <img className={`food ${disabled ? "disabled" : ""}`} src={require(`../images/${foodOption}.png`)} alt={foodOption} />
+        <button className={`${styles.button} ${disabled ? "disabledButton" : ""}`} disabled={disabled} onClick={() => onClick(foodOption)}>
+            <img className={`${styles.food} ${disabled ? "disabled" : ""}`} src={require(`../images/${foodOption}.png`)} alt={foodOption} />
```

Comment:
`"disabled"` can be replaced with `styles.disabled` if you move those classes into the module.

Reply:
same with `disabledButton`

Reply:
Nice catch thanks :)